lib/rucio/web/rest/flaskapi/v1/dids.py

```diff
@@ -575,9 +576,15 @@ def get(self, scope_name):
           schema:
             type: string
           style: simple
+        - name: dynamic_depth
+          in: query:
+          description: The DID type at which to stop the dynamic length/size estimation
+          schema:
+            type: string
+            enum: ["FILE", "DATASET"]
         - name: dynamic
           in: query
-          description: Flag to dynamically calculate size for open DIDs.
+          description: (Deprecated) Same as dynamic_depth = "FILE"
```

Comment:
This is actually a premiere event for us. The `OpenApi` specification has a `deprecated` attribute (https://swagger.io/specification/#parameter-object).
```text
description: Same as dynamic_depth = "FILE".
deprecated: true
```